tests/nested/manual/passphrase-support-on-hybrid/task.yaml

```diff
@@ -0,0 +1,196 @@
+summary: End-to-end test for FDE passphrase support on hybrid systems
+
+details: |
+  This test installs an encrypted hybrid Ubuntu system using muinstaller
+  which is protected by passphrase authentication.
+
+systems: [ubuntu-24.04-64]
+
+# TODO: Remove this when passphrase support lands.
+manual: true
+
+environment:
+  NESTED_ENABLE_TPM: true
+  NESTED_ENABLE_SECURE_BOOT: true
+
+  # TODO: Swap 24.10 with 25.04 when the kernel/gadget snaps are available in the store.
+  # Mimic default channels specified in https://github.com/canonical/models/blob/master/ubuntu-classic-2504-amd64-dangerous.json.
+  GADGET_CHANNEL: classic-24.10/edge
+  KERNEL_CHANNEL: 24.10/edge
+  CORE_VERSION: 22
+
+  PASSPHRASE: ubuntu
+  # "pbkdf2" is less memory intensive than argon2i* so we use it by default.
+  # TODO: Add argon2i* variants.
+  KDF_TYPE: pbkdf2
+
+  # Ensure we use our latest code.
+  NESTED_BUILD_SNAPD_FROM_CURRENT: true
+  NESTED_REPACK_KERNEL_SNAP: true
+  NESTED_ENABLE_OVMF: true
+  # Store related setup.
+  STORE_ADDR: localhost:11028
+  STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+  if [ "$TRUST_TEST_KEYS" = "false" ]; then
+      echo "This test needs test keys to be trusted"
+      exit
+  fi
+  apt install dosfstools kpartx
```

Comment:
Please add a comment about what uses those.

Because specially I am not fan of kpartx, and I would like to know if we can just remove it.

Reply:
can you move dosfstools to the prepare section of tests/nested/manual in spread.yaml? FWIW, those should already be present anyway, so I'm not sure why we need to install them explicitly, perhaps it works without the package too?

As for kpartx it's likely some legacy thing that's incorrectly perpetuated, the prepare sections of tests/nested in spread.yaml list it, but AFAIK we do not need it, neither does ubuntu-image, so if possible we could use it as an opportunity to remove it.

Reply:
kpartx is used inside `tests/lib/tools/setup_nested_hybrid_system.sh`, also `fatlabel` which is provided by the `dosfstools` package is also used inside `tests/lib/tools/setup_nested_hybrid_system.sh`.

I will try to move them up into prepare of `tests/nested/manual`

Reply:
turns out kpartx is already installed in nested prepare for all variants